docs/USING.rst

```diff
@@ -111,19 +111,94 @@ For nodes using IPMI use the following api call:
 
 ::
 
-   curl -X PUT http://127.0.0.1:5001/node/dummyNode01 -d '
-   > {"obm": { "type": "http://schema.massopencloud.org/haas/v0/obm/ipmi",
-   > "host": "ipmiHost4node-01",
-   > "user": "ipmiUser4node-01",
-   > "password": "ipmiPass4node-01"
-   > }}'
+     curl -X PUT http://127.0.0.1:5001/node/dummyNode01 -d '
+     > {"obm": { "type": "http://schema.massopencloud.org/haas/v0/obm/ipmi",
+     > "host": "ipmiHost4node-01",
+     > "user": "ipmiUser4node-01",
+     > "password": "ipmiPass4node-01"
+     > }}'
 
 Corresponding cli calls will be as follows:
 
 
 ::
 
-        hil node_register ipmi dummyNode01 ipmiHost4node-01 ipmiUser4node-01 ipmiPass4node-01
+     $ hil node_register ipmi dummyNode01 ipmiHost4node-01 ipmiUser4node-01 ipmiPass4node-01
+
+3) Creating a Project in HIL
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+Eg> Project name mockproject01
+
+api call
+
+
+::
+
+    curl -X put http://127.0.0.1:5000/project/mockproject01
+
+cli call
+
+::
+
+
+    $ hil project_create mockproject01
+
+4) Register a node in HIL
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+Eg> Node name: mocknode01
```

Comment:
This doesn't render particularly well; all of the fields get wrapped on to one line. Also the `Eg>` notation is weird to me.

Reply:
Hmm, I am trying to follow the previous examples. And we actually have two different formats in those two existing examples. I am gonna reformat them then and try to come up with one clean format.

Reply:
I hadn't noticed it was existing notation, but yeah, let's improve upon it.